app/scripts/controllers/alert-controller.ts

```diff
@@ -76,12 +80,15 @@ export type AlertControllerState = {
  * @property state - The initial controller state
  * @property controllerMessenger - The controller messenger
  */
-type AlertControllerOptions = {
-  state?: Partial<AlertControllerState>;
-  controllerMessenger: AlertControllerMessenger;
+export type AlertControllerOptions = {
+  state: Partial<AlertControllerState>;
```

Comment:
I didn't catch this earlier, but conventionally we make the `state` property optional:

``` suggestion
  state?: Partial<AlertControllerState>;